reconfiguration/src/reconfiguration_handler.cpp

```diff
@@ -57,6 +57,15 @@ bool ReconfigurationHandler::handle(const KeyExchangeCommand& command,
 
   return true;
 }
+bool ReconfigurationHandler::handle(const concord::messages::AddRemoveWithWedgeCommand&,
+                                    uint64_t bft_seq_num,
+                                    concord::messages::ReconfigurationResponse&) {
+  LOG_INFO(getLogger(), "AddRemoveWithWedgeCommand instructs replica to stop at sequence number " << bft_seq_num);
+  bftEngine::ControlStateManager::instance().setStopAtNextCheckpoint(bft_seq_num);
+  bftEngine::IControlHandler::instance()->addOnStableCheckpointCallBack(
```

Comment:
Following the "adding bft field to the command", if we have bft=false, then the callback should be called on superStableCheckpoint (see the wedge handler)

Reply:
fixed